R/datasets.R

```diff
@@ -362,3 +362,20 @@
 #' }
 #' @seealso This data was sampled from the `movies` data frame in the `ggplot2movies` package.
 "movies_sample"
+
+
+#' Massachusetts 2020 vs. 2019 Traffic Data Comparison
+#'
+#' Traffic data comes from two primary sources:
+#' <https://massdot-impact-crashes-vhb.opendata.arcgis.com/datasets/MassDOT::2020-vehicle-level-crash-details/explore> and 
+#' <https://mhd.public.ms2soft.com/tcds/tsearch.asp?loc=Mhd&mod=>
+#' 
+#' @format A data frame of 264 rows representing a different community in Massachusetts.
+#' \describe{
+#'   \item{community}{City or Town}
+#'   \item{functional Class}{Class or group the road belongs to}
+#'   \item{change_in_Speed}{Average estimated Speed (mph)}
+#'   \item{change_in_Volume}{Average traffic}
+#'   \item{change_in_Accidents}{Average number of accidents}
+#' }
+"ma_traffic_2020_vs_2019"
```

Comment:
Please also commit the changes to the `man/ma_traffic_2020_vs_2019.md` file

Reply:
I don't see any changes to commit for this. For some reason the documentation for this data set is not registering there, and I don't know why. @beanumber

Reply:
I added the changes to `man/ma_traffic_2020_vs_2019.md` . My computer automatically updated the `RoxygenNote: 7.1.1` version. I hope this is okay....